src/app/Http/Controllers/V1/EcoKirimNaskahController.php

```diff
@@ -0,0 +1,156 @@
+        $nama_pejabat = $request->nama_pejabat;
+        $jabatan = $request->jabatan;
+        $file = $request->file;
+        $inboxId              = $unit_pencipta . date('dmyhis');
+        $tableKey             = TableSetting::first()->tb_key;
+
+        $infoPengirim = People::select('rolecode.rolecode_name as instansi_pengirim','people.PeopleName as people_name',
+                    'rolecode.rolecode_id as instansi_pengirim_id','role.RoleId as jabatan_id',)
+                    ->join('role', 'role.RoleId', '=', 'people.PrimaryRoleId')
+                    ->join('rolecode', 'rolecode.rolecode_id', '=', 'role.RoleCode')
+                    ->where('peopleId','=',$unit_pencipta)->first();
+        
+        $infoPenerima = People::select('people.PeopleName as people_name','role.RoleId as jabatan_id')
+                    ->join('role', 'role.RoleId', '=', 'people.PrimaryRoleId')
+                    ->join('rolecode', 'rolecode.rolecode_id', '=', 'role.RoleCode')
+                    ->where('peopleId','=',$tujuan)->first();
+        
+        // INSERT INBOX 
+        $inbox = new Inbox();
+        $inbox->setRawAttributes([
+            'NTglReg' => $tanggal_persetujuan,
+          ]);
+        $inbox->NKey            = $tableKey;
+        $inbox->NId             = $inboxId;
+        $inbox->CreatedBy       = $unit_pencipta;
+        $inbox->CreationRoleId  = $infoPengirim->jabatan_id;
+        $inbox->Tgl             = $tanggal;
+        $inbox->JenisId         = $jenis_naskah;
+        $inbox->UrgensiId       = 'XxJyPn38Yh.1';
+        $inbox->SifatId         = $sifat_naskah;
+        $inbox->Hal             = $hal;
+        $inbox->Nomor           = $nomor_naskah;
+        $inbox->Pengirim        = 'eksternal';
+        $inbox->NTipe           = 'inbox';
+        $inbox->Namapengirim    = $nama_pejabat;
+        $inbox->NFileDir        = 'naskah';
+        $inbox->BerkasId        = '1';
+        $inbox->Instansipengirim   = $infoPengirim->instansi_pengirim;
+        $inbox->InstansiPengirimId = $infoPengirim->instansi_pengirim_id;
+        $inbox->isi_naskah      = $isi;
+        $inbox->ClId            = $kode_klasifikasi;
+        $inbox->JabatanPengirim = $jabatan;
+        $inbox->Kepada_Yth      = $infoPenerima->people_name;
+        $inbox->tembusan        = $tembusan;
+        //dd($inbox);exit();
+        $inbox->save();
+
+        // INSERT INBOX RECEIVER
+        $dataInboxReceiver = [
+            'NId'           => $inboxId,
+            'NKey'          => $tableKey,
+            'GIR_Id'        => $inboxId,
+            'From_Id'       => $unit_pencipta,
+            'RoleId_From'   => $infoPengirim->jabatan_id,
+            'To_Id'         => $tujuan,
+            'RoleId_To'     => $infoPenerima->jabatan_id,
+            'Msg'           => '',
+            'StatusReceive' => 'unread',
+            'ReceiveDate'   => date('Y-m-d H:i:s'),
+            'action_label'  => ActionLabelTypeEnum::REVIEW(),
```

Comment:
punten ini ternyata ada yang kurang kang @syaifulmuflih95 kang @bioe56 , yaitu field `ReceiverAs` dengan value `to`. 
```suggestion
            'action_label'  => ActionLabelTypeEnum::REVIEW(),
            'ReceiverAs'  => 'to'
```